kits/why-this-code/README.md

```
@@ -0,0 +1,121 @@
+npm run dev
+```
+
+Open [http://localhost:3000](http://localhost:3000) and paste a supported GitHub permalink (e.g. `https://github.com/owner/repo/blob/main/src/service.ts#L10`).
+
+#### 3. Deploy on Vercel (optional)
+
+Deploy the `apps` directory to Vercel and configure the 4 environment variables above (`LAMATIC_API_URL`, `LAMATIC_PROJECT_ID`, `LAMATIC_API_KEY`, `WHY_THIS_CODE`).
+
+#### 4. Example
+
+Sample Input: [here](./sample_input.json)
```

markdownlint-cli2 (0.23.0)

[warning] Link text should be descriptive

(MD059, descriptive-link-text)